blueocean-dashboard/src/main/js/components/PipelineRunGraph.jsx

```diff
@@ -173,7 +182,7 @@ export default class PipelineRunGraph extends Component {
 PipelineRunGraph.propTypes = {
     pipelineName: string,
     branchName: string,
-    runId: string,
```

Comment:
I created https://issues.jenkins-ci.org/browse/JENKINS-38482 and assigned to @michaelneale to get the ball rolling. ...„Der Ball muß ins Eckige“. ;)